hana/monitoring.sls

```diff
@@ -31,7 +35,7 @@ install_python_pip:
 extract_pydbapi_client:
   hana.pydbapi_extracted:
     - name: PYDBAPI.TGZ
-    - software_folders: [{{ exporter.hana_client_path|default(node.install.software_path)|default(hana.software_path)|default(hana_extract_dir) }}]
+    - software_folders: [{{ exporter.hana_client_path|default(node.install.software_path)|default(hana.software_path)|default(hana_client_path) }}]
```

Comment:
I guess we will need to remove the `hana_client_path` from `exporter` entry, right?
They seem duplicated

Reply:
We can leave it to provide node specific `hana_client_path` right